### ISSUE TITLE
Module and Box libraries support

### ISSUE THREAD

**Issue by latot** [r-lib/lintr#2060]
Hi all!

I was using lintr from VSCode, then I notice.... the operator ```%.>%``` from wrapr was detected a no visible global variable.

Then I test adding to the module ```library(wrapr)``` and then the warning disappears.

Actually the code is inside a module, there is two main libs to works with modules in R, Modules and Box, which are great to finally organize the code.

From modules we cannot use ```library``` we use other operators.

For modules, ```modules::import("wrapr")```

For box: ```box::use("wrapr")```,  to import all ```box::use(ggplot2[...])```, to import one function ```box::use(ggplot2[ggplot])```

That is the way to import functions and operators, would be great be able to detect them!

Thx!

**Comment by radbasa** [comment]
We recently released [`{box.linters}`](https://cran.r-project.org/package=box.linters). Please take a look. We would appreciate feedback.
